### ISSUE TITLE
Remove tumblr links from the app

### ISSUE THREAD

**Issue by DeeDeeG** [RefugeRestrooms/refugerestrooms#429]
Here's what I get when I visit https://blog.refugerestrooms.org/

![wrong_ssl_certificate](https://user-images.githubusercontent.com/20157115/35165463-5ee2beac-fd1c-11e7-9ada-5ba9f51ea05e.png)

**Comment by stardust66** [comment]
Probably a setting in tumblr.

**Comment by DeeDeeG** [comment]
Yes, this does appear to be a quick settings change over at tumblr.

> Update to the update: Now SSL is available for blogs with custom domains, too. To turn it on, go to your [blog settings](https://www.tumblr.com/settings/blog) and enable “Always serve blog over SSL.” Once you’ve done that, it takes a while (typically less than a day) for the SSL on custom domains to activate. We’ll send you an email when it’s ready. 

From: https://security.tumblr.com/post/166219476810/support-support-ssl-security-which-has-been

**Comment by mi-wood** [comment]
@tkwidmer if you share the tumblr login with me, I'll fix this

**Comment by tkwidmer** [comment]
In light of Tumblr's recent policy changes that really position them against sex positive / LGBTQ communities, I think we should look to a different source for this entirely.

**Comment by tkwidmer** [comment]
The tumblr is on my account, which i'm in the process of closing out.

**Comment by mi-wood** [comment]
We can utilize our github.io for this probably and delete tumblr from our link section. Going to update this ticket to reflect that.

**Comment by mi-wood** [comment]
Note, we should also update the DNS setting to not point to tumblr...

**Comment by DeeDeeG** [comment]
Someone seems to have parked the tumblr blog (and kept it empty/devoid of posts). As far as I can tell this not someone directly working on the app (part of the addmitedly informal "team.")

It's nice of someone to look out for us by parking the name, but I think we are assuming their good graces etc. (They could decide to post something at some point.)

tl;dr https://blog.refugerestrooms.org/ points to an empty tumblr page (apparently not owned by us).
